crates/shared/src/utils.rs

```diff
@@ -323,6 +343,46 @@ mod tests {
             .await
             .expect_err("API is reachable, but is on wrong path");
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn event_stream_wrapper_with_idle_timeout() {
```

Comment:
Nit: Can we add `test_` to the test name to distinguish it from test helpers? Would be nice to rename other tests in this file as well if not too much additional work!

Reply:
[9561de9](https://github.com/EspressoSystems/marketplace-builder-core/pull/257/commits/9561de982902417d0821767bf71988c8e0a7e324).